docs/api/storage.yml

```diff
@@ -118,6 +127,9 @@ paths:
       produces:
         - application/octet-stream
         - application/json; charset=utf-8
+        - text/openEhrXml
+        - text/openEhrJson
```

Comment:
why are these two type listed here?

Reply:
Hm, I've added it here cause this endpoint will produce this content-types. Afaik go-swagger generated server will overwrite content-type with application/octet-stream if it's not set here. Of course default producers can be also changed in code but decided that it's cleanest to just add it here. Let me know what you think.

Reply:
ok